samples/bluetooth/iso_broadcast/overlay-bt_ll_sw_split.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_BT_CTLR_ADV_ISO=y
+CONFIG_BT_CTLR_ADV_ISO_PDU_LEN_MAX=4
+CONFIG_BT_CTLR_ADV_ISO_STREAM_MAX=2
+
+CONFIG_BT_CTLR_ISOAL_SOURCES=2
+
+# FIXME: Host needs BT_ISO_TX_MTU + 4 bytes otherwise it tries to fragment ISO
+# data. When Host is fixed, BT_CTLR_ISO_TX_BUFFER_SIZE can inherit BT_ISO_TX_MTU
+# value.
```

Comment:
Or 8 if using timestamp, right?

Reply:
I do not know when Host uses timestamp, but currently for this sample +4 seems to be sufficient.

Reply:
This sample uses `BT_ISO_TIMESTAMP_NONE` when sending, so that will work. 

But if someone modifies the call to `bt_iso_chan_send`, then 4 octets may not be enough. 

Not something that needs to change in your PR though, but something to keep in mind